wiki/en/misc/1-index.html

```diff
@@ -17,7 +17,7 @@
   <div class="fx-col-100-xs">
     <div itemprop="abstract">
       <h2>What is Jamulus?</h2>
-       Jamulus is for playing, rehearsing, or just jamming with your friends, your band or just anyone you find online. With high quality, low-latency sound on a normal broadband connection, it's easy to play together remotely and in time. <a href="wiki/Getting-Started" target="_blank" rel="noreferrer">Download it here</a>!
+       Jamulus is a software for playing, rehearsing, or jamming with your friends, your band or anyone you find online. Play together remotely in time with high quality, low-latency sound on a normal broadband connection. <a href="wiki/Getting-Started" target="_blank" rel="noreferrer">Download it here</a>!
```

Comment:
`a software` is idiomatically incorrect English.  Just say "Jamulus allows playing ..."?